themes/frontierline/functions.php

```diff
@@ -723,4 +727,13 @@ function save_frontierline_display_hero() {
 
 add_action('save_post', 'save_frontierline_display_hero');
 
+// hide hero image uploaod box if they're disabled 
+function remove_featured_box() {
```

Comment:
I would consider keeping the upload box there. The featured image is still used in meta https://github.com/craigcook/frontierline-theme/blob/master/themes/frontierline/includes/metadata.php#L32

Reply:
I'll defer to @craigcook on this. What say you Craig?